extensions/wikia/ArticleVideo/bower_components/html5-skin/js/controller.js

```diff
@@ -1294,8 +1319,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
             "showVideoQualityPopover":this.state.videoQualityOptions.showVideoQualityPopover
           }
         });
-      if(this.state.videoQualityOptions.showVideoQualityPopover == true) {
-        this.toggleVideoQualityPopOver();
+      if(this.state.configPanelOptions.showConfigPanelPopover == true) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace "==" with "===". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AEqEqEq)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1125)